tests/main/views/test_suppliers.py

```diff
@@ -1914,6 +1914,57 @@ def test_set_application_company_details_confirmed_updates_declaration(
         }
 
 
+class TestDeleteUnsuccessfulApplicantDeclarations(BaseApplicationTest, FixtureMixin,
+                                                  PutDeclarationAndDetailsAndServicesMixin):
+    def setup(self):
+        """
+        This sets up the class. It creates a dummy framework and a supplier that applied to that framework, with a
+        declaration.
+        :return:
+        :rtype:
+        """
+        super(TestDeleteUnsuccessfulApplicantDeclarations, self).setup()
+
+        self.supplier_id = self.setup_dummy_suppliers(1)[0]
+        self.framework_slug = 'digital-outcomes-and-specialists'
+        self.set_framework_status('digital-outcomes-and-specialists', 'open')
+        self.updater_json = {'updated_by': 'Joe Bloggs'}
+        self._register_supplier_with_framework()
+
+    def test_a_supplier_framework_object_is_returned_with_an_empty_declaration(self):
+        self._put_declaration("Success!")
```

Comment:
I know I suggested this delightful string but we should probably use a valid declaration status (`started` or `complete`).

Reply:
I also wonder if we should test what happens if the declaration is already empty, e.g. if a careless dev hits the endpoint twice (hopefully the endpoint won't care?)